aspnet/fundamentals/environments.rst

```diff
@@ -94,6 +94,8 @@ ASP.NET Core supports a convention-based approach to configuring an application'
 
 When an ASP.NET Core application starts, the ``Startup`` class is used to bootstrap the application, load its configuration settings, etc. (:doc:`learn more about ASP.NET startup <startup>`). However, if a class exists named ``Startup{EnvironmentName}`` (for example ``StartupDevelopment``), and the ``ASPNETCORE_ENVIRONMENT`` environment variable matches that name, then that ``Startup`` class is used instead. Thus, you could configure ``Startup`` for development, but have a separate ``StartupProduction`` that would be used when the app is run in production. Or vice versa.
 
+.. note:: If you have a call to ``WebHostBuilder.UseStartup<TStartup>()``, it will override configuration sections.
```

Comment:
+.. note:: If you ~~have a~~ call ~~to~~ ``WebHostBuilder.UseStartup<TStartup>()``, it will override configuration sections. 
Or
Calling ``WebHostBuilder.UseStartup<TStartup>()``  overrides configuration sections.